ffmpeg_plugin/README.md

```diff
@@ -1,32 +1,78 @@
 # svt-vp9 ffmpeg plugin installation
 
-1. Build and install SVT-VP9
-- git clone https://github.com/OpenVisualCloud/SVT-VP9
-- cd SVT-VP9
-- cd Build && cmake .. && make -j `nproc` && sudo make install
-
-2. Apply SVT-VP9 plugin and enable libsvtvp9 to FFmpeg
-- git clone https://github.com/FFmpeg/FFmpeg ffmpeg
-- cd ffmpeg
-- git checkout -b tag4.2.2 n4.2.2
-- export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:/usr/local/lib
-- export PKG_CONFIG_PATH=${PKG_CONFIG_PATH}:/usr/local/lib/pkgconfig
-- SVT-VP9 alone:
-   - git apply SVT-VP9/ffmpeg_plugin/0001-Add-ability-for-ffmpeg-to-run-svt-vp9.patch
-   - ./configure --enable-libsvtvp9
-- Based on SVT-HEVC & SVT-AV1:
-   - git apply SVT-HEVC/ffmpeg_plugin/0001-lavc-svt_hevc-add-libsvt-hevc-encoder-wrapper.patch
-   - git apply SVT-AV1/ffmpeg_plugin/0001-Add-ability-for-ffmpeg-to-run-svt-av1-with-svt-hevc.patch
-   - git apply SVT-VP9/ffmpeg_plugin/0001-Add-ability-for-ffmpeg-to-run-svt-vp9-with-svt-hevc-av1.patch
-   - ./configure --enable-libsvthevc --enable-libsvtav1 --enable-libsvtvp9
-- make -j $(nproc)
-
-- Note: patches for FFmpeg n4.2.3/master are alse provided:
-   - n4.2.3-0001-Add-ability-for-ffmpeg-to-run-svt-vp9.patch
-   - n4.2.3-0001-Add-ability-for-ffmpeg-to-run-svt-vp9-with-hevc-av1.patch
-   - master-0001-Add-ability-for-ffmpeg-to-run-svt-vp9.patch
-   - master-0001-Add-ability-for-ffmpeg-to-run-svt-vp9-with-hevc-av1.patch
-   
-3. Verify
-- ./ffmpeg  -i input.mp4 -c:v libsvt_vp9 -rc 1 -b:v 10M -preset 1  -y test.ivf
-- ./ffmpeg  -i input.mp4 -vframes 1000 -c:v libsvt_vp9 -y test.mp4
+Assumes you have a bash (or similar) shell along with commands such as
+
+- git
+- cmake
+- make
+- pkg-config
+- cc
+
+If you do not have any of these, please consult your friendly web browser to determine what is the best way to get these on your system, else ask IT.
+
+If you are on windows, the recommended way to get these tools and build FFmpeg is through [msys2](https://www.msys2.org/). The way to get these specific tools will not be provided here though.
+
+If you are trying to compile FFmpeg using MSVC, please try to consult FFmpeg's [wiki page](https://trac.ffmpeg.org/wiki/CompilationGuide/MSVC) about it.
+
+## Build and install SVT-VP9
+
+```bash
+git clone https://github.com/OpenVisualCloud/SVT-VP9.git
+cd SVT-VP9/Build
+# Adjust this line to add any other cmake related options
+cmake .. -DCMAKE_BUILD_TYPE=Release
+make -j $(nproc)
+sudo make install
+```
+
+## Apply SVT-VP9 plugin and enable libsvtvp9 to FFmpeg
+
+beginning commands:
+
+```bash
+# Go back to the repo root dir
+cd ..
```

Comment:
```
# Go back to the repo root dir
cd ..
```
Is this necessary to mention?

Reply:
This is to make sure that the person the paths won't be messed up in case the person decides to build in another dir, not necessary, but would help in the case there's someone who might not understand basic instructions.

This is mainly for the patch paths.